src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, onClose }) => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [loading, setLoading] = useState(true);
```
to remove blinking when first time open the modal